starter-code/index.html

```diff
@@ -9,6 +9,50 @@
   <script type="text/javascript" src="./js/index.js"></script>
   <title>Ironhack cart</title>
 </head>
-<body>
-</body>
+
+<!-- <body>
+  <h1>Ironhack <br> Merchandising Shop</h1>
+  <div>
+    <div>
+      <input type="text" class="product-name">
+    </div>
+    <div>
+        <input type="number" class="unit-cost">
+    </div>
+    <div>
+      <label>QTY</label>
+      <input type="number" class="quantity">
+    </div>
+    <div>
+        <span class="row-price">$0.00</span>
+    </div>
+    <div>
+        <button>Calculate Prices</button>
+    </div>
+  </div>
+</body>-->
+<body> 
+    <h1>Ironhack <br> Merchandising Shop</h1>
+    <div>
+      <div>
+        <span class="product-name">IronBabble-head</span>
+      </div>
+      <div>
+          <span class = "unit-cost">25.00</span>
+      </div>
+      <div>
+        <label>QTY</label>
+        <input type="number" class="quantity">
+      </div>
+      <div>
+          <span>$0.00</span>
+      </div>
+      <div>
+          <button>Delete</button>
+      </div>
+      <div>
+          <button>Calculate Prices</button>
```

Comment:
There's an id missing here that would connect the getTotalPrice function with this one.
```var calculatePriceButton = document.getElementById('calc-prices-button');```